Detectors/TRD/calibration/include/TRDCalibration/KrCalibUploadToCCDB.h

```diff
@@ -31,20 +31,19 @@ namespace o2
 namespace trd
 {
 
-class KrCalibration
+class KrCalibUploadToCCDB
```

Comment:
Actually, the name is still misleading: this class does not upload to the CCDB, it prepares the CCDB object. A better name might be something like PadCalibCCDBBuilder.
@bazinski, @martenole  is there some convention on naming such classes?

Reply:
Hello. I think we are close to merging this PR. Should we invent a new name, or stick with the current one? @martenole , @bazinski ?